salt/modules/glusterfs.py

```diff
@@ -238,6 +238,12 @@ def create_volume(name, bricks, stripe=False, replica=False, device_vg=False,
         Replica count, the number of bricks should be a multiple of the \
         replica count for a distributed replicated volume
 
+    arbiter
+        If true, specifies volume should use arbiter brick(s). \
+        Valid configuration limited to "replica 3 arbiter 1" per \
+        Gluster documentation. Every third brick in the brick list \
+        is used as an arbiter brick.
+
```

Comment:
Can you add a `.. versionadded:: Fluorine` tag to these new docs?

Reply:
Done-ish? I wasn't 100% sure on the proper syntax for that tag. I added it just above the new documentation I added, but it looks slightly janky. Let me know if it's OK as-is, or if it should be done differently.

Reply:
Oh, you're close. It needs to be below the arg name. Something like this:

```
arbiter
    words words words

    .. versionadded:: Fluorine
```

Reply:
Ah! That looks much better. I tried putting it under initially, but didn't have a blank line after the "words words words" bit, so it didn't interpret it as a tag. Fixed now, thanks!

Reply:
Sure thing! Thanks for fixing this up so quickly!